python/ray/train/constants.py

```diff
@@ -78,11 +78,16 @@ def _get_ray_train_session_dir() -> str:
 # Defaults to 0, which always retries on node preemption failures.
 RAY_TRAIN_COUNT_PREEMPTION_AS_FAILURE = "RAY_TRAIN_COUNT_PREEMPTION_AS_FAILURE"
 
+# Set this to 1 to start a StateActor and collect information Train Runs
+# Defaults to 0
+ENABLE_TRAIN_RUN_STATE_TRACKING_ENV = "ENABLE_TRAIN_RUN_STATE_TRACKING_ENV"
```

Comment:
Nit: No need for the `ENV` suffix in the actual env var string. Also, let's have the prefix for our env variables start with `RAY_TRAIN`:

`RAY_TRAIN_ENABLE_STATE_TRACKING`

Reply:
Good point.